src/mlpack/tests/rl_components_test.cpp

```diff
@@ -87,18 +88,34 @@ BOOST_AUTO_TEST_CASE(RandomReplayTest)
   BOOST_REQUIRE_EQUAL(1, replay.Size());
 
   //! Overwrite the memory with a nonsense record
-  for (size_t i = 0; i < 5; ++i) {
+  for (size_t i = 0; i < 5; ++i)
     replay.Store(nextState, action, reward, state, true);
-  }
+
   BOOST_REQUIRE_EQUAL(3, replay.Size());
 
   //! Sample several times, the original record shouldn't appear
-  for (size_t i = 0; i < 30; ++i) {
+  for (size_t i = 0; i < 30; ++i)
+  {
     replay.Sample(sampledState, sampledAction, sampledReward, sampledNextState, sampledTerminal);
     CheckMatrices(state.Encode(), sampledNextState);
     CheckMatrices(nextState.Encode(), sampledState);
     BOOST_REQUIRE_EQUAL(true, arma::as_scalar(sampledTerminal));
   }
 }
 
+/**
+ * Construct a greedy policy instance and check if it works as
+ * it should be.
+ */
+BOOST_AUTO_TEST_CASE(GreedyPolicyTest)
+{
+  GreedyPolicy<CartPole> policy(1.0, 10, 0.0);
+  for (int i = 0; i < 15; ++i)
```

Comment:
Do you mind to use ``size_t`` instead of ``int`` here, I know it is kind of a pedantic request.